bundles/http/Citrix/2ec16a1b-9423-45a6-8fd8-b0c42c916c0f/cb5694f4-3740-45d6-9817-44d451edb2d9.js

```diff
@@ -0,0 +1,1136 @@
+
+//Defined meetingarray to store meeting id, while will be used in meeting details api call
+let meetingArray = [];
+
+// Defined the Map for storing the meetingid as key and user email as value, which is used in meeting invitations api call 
+let meetingsMap = new Map();
+
+
+
+// loading the users data
+function syncUsers(dataStore,client,context){
+    let userContext = [];
+    do{
+        let request = client.fetchSync(`/users?next_page_token=${user.next_page_token}&page_size=${user.page_size}`);
+        // console.log(`users status : ${request.status}`);
+        if(!request.ok){
+            throw new Error(request.jsonSync());
+        }
+        let response = request.jsonSync();
+        Object.entries(response).forEach(([key,value]) =>{
+            if(key === 'next_page_token'){
+                user.next_page_token=value // 
+            }
+            else if(key === 'users'){
+                let usersData = value.map(userData =>{
+                    userContext.push(userData.id);
+                    let userObj = {
+                        "userid":userData.id,
+                        "first_name":userData.first_name,
+                        "last_name":userData.last_name,
+                        "email":userData.email,
+                        "timezone":userData.timezone,
+                        "last_login_time":moment.utc(userData.last_login_time).toDate()
+                    }
+                    userids.push(userObj); //storing the user data in array to be used in child endpoints
+                    return userObj;
+                })
+                dataStore.save("users",usersData);
+            }
+        });
+    }while(user.next_page_token!="");
+    context.user = userContext.toString();
+}
+
+//loading the meetings data (Upcoming Meetings)
+function syncMeetings(dataStore,client,userId=[]){
+    let i = 0;
+    let array = [];
+    //definig login to distinguish between fullsync and data update after action, as userId array is passed while performing data update after action
+    if(userId.length>0){
+       array = userId;
+    }
+    else{
+        array = userids;
+    }
+    do{
+        let response = client.fetchSync(`users/${array[i].userid}/meetings?type=upcoming&next_page_token=${meeting.next_page_token}&page_size=${meeting.page_size}`);
+        // console.log(`${userids[i]} status : ${response.status}`);
+        if(!response.ok){
+            throw new Error(response.jsonSync());
+        }
+        let responseData = response.jsonSync();
+        meeting.next_page_token = responseData['next_page_token'];
+        let meetings = responseData.meetings;
+        let data = meetings.map(meeting =>{
+            if(userId.length == 0){
+                meetingArray.push(meeting.id);
+            }
+            else{
+                meetingArray.push(array[i].meetingid)
+            }
+            return {
+                "uuid":meeting.uuid,
+                "id":Number(meeting.id),

```

Comment:
Just curious: why do you create `Number` object? 
IMHO `meeting.id` is already `typeof "number"`